.github/workflows/update-providers-auto.yml

```
@@ -2,9 +2,9 @@ name: Update Providers with new bridge version upon release
 on:
   push:
     tags:
-      # Typically pf* module is tagged after the main module, so this is a good time to fire off
-      # automatic downstream upgrades.
-      - 'pf/**'
+      # Automatically trigger on valid patch releases of the bridge.
+      - v*.*.*
+      - '!v*.*.*-**' # Do not propagate prereleases
```

It feels like we should quote these equally:
```suggestion
      - "v*.*.*"
      - "!v*.*.*-**" # Do not propagate prereleases
```